ZelBack/src/services/fluxNetworkHelper.js

```diff
@@ -287,12 +212,9 @@ async function isFluxAvailable(ip, port = config.server.apiport) {
     if (!UIok) return false;
 
     const syncthingPort = +port + 2;
-    // eslint-disable-next-line no-use-before-define
-    const syncthingOpen = await isPortOpen(ip, syncthingPort);
-    if (!syncthingOpen) return false;
-
-    return true;
+    return isPortOpen(ip, syncthingPort);
```

Comment:
There is no await here. So this will practically return a promise
should be 
const result = awaits isPortOpen
return result;

Reply:
I believe when you want to return a async function, it will await by default, I think it's correct the way it is, what you are suggesting it's the same writting in a different way.